Make snake game server address configurable

lenkan opened the pull request:
This PR fixes #6

emilb commented:
Okay, good work on config! 

It seems unconventional to me that the whole URL is passed as a config parameter. The parameter now has a default value of: http://snake.cygni.se/events and that is what would be passed as a parameter to the service on startup (if differing from default of course). 

Conventional (to me) would be to only pass the hostname as a parameter. I think it is reasonable to assume that the protocol will always be http and the URI will always be events.

If the reason for using only one configuration parameter is that the port (:8080 for local) comes in to play I'd prefer that it was it's own configuration parameter. Looking at other projects and similar setups this is what I've seen. 

What do you think?

lenkan commented:
I addressed your comments by splitting the configuration into a 'server-host' option and a 'server-port' option. Example "npm run build -- --server-host snake.cygni.se" or "npm run build -- server-host "localhost" --port 8080". I could not use --host and --port because they are used by webpack-dev-server. Merge?

johannesdolk commented:
Good work!
